src/backend/joanie/settings.py

```diff
@@ -275,6 +275,20 @@ class Base(Configuration):
         environ_prefix=None,
     )
 
+    JOANIE_LCMS_BASE_URL = values.Value(
+        environ_name="JOANIE_LCMS_BASE_URL",
+        environ_prefix=None,
+    )
+
+    # Context processors for document issuers
+    JOANIE_DOCUMENT_ISSUER_CONTEXT_PROCESSORS = {
+        "contract_definition": values.ListValue(
+            [],
```

Comment:
Why not adding the `richie_syllabus` processor here ?

Reply:
Because, from my point of view, this is something specific to our use case. I do not want to enable this feature by default.